rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/380_sort_segments_on_timestamp.yml

```diff
@@ -15,7 +15,6 @@
                 type: date
           settings:
             number_of_shards: 1
-            number_of_replicas: 0
```

Comment:
Last I checked these tests were making assumptions about refreshes and the corresponding number of underlying segments. Is that no longer the case?